src/app/pages/properties/properties.component.ts

```diff
@@ -0,0 +1,47 @@
+import { PropertiesService } from './../../shared/services/properties/properties.service';
+import { Component, OnInit } from "@angular/core";
+import { Property } from "../../models/Property";
+
+@Component({
+  selector: "app-properties",
+  templateUrl: "./properties.component.html",
+  styleUrls: ["./properties.component.scss"]
+})
+export class PropertiesComponent implements OnInit {
+  properties: Property[];
+  propertiesUrl = "https://landville-backend-web-api.herokuapp.com/api/v1/properties/";
+  next: string;
+  previous: string;
+  toggle: boolean = true;
+
+  constructor(private propertiesServices: PropertiesService) {}
+
+  ngOnInit(): void {
+    this.setProperties(this.propertiesUrl);
+  }
+
+  setProperties(url: string) {
```

Comment:
I noticed that when there are no property to display, the site almost looks broken ... Maybe you could add some information notifying the users that there are no property to display and they could check back later?

Reply:
Thanks @MandelaK , I have worked on this.

Reply:
Hey @Genza999, on the same note when there are no properties, The footer is way in the middle of the page which does not look great,we could try and make it stick to the bottom.

Reply:
Hey @CryceTruly Thanks I have worked on this issue.